docs/src/guides/integration/create-identity.md

```diff
@@ -0,0 +1,59 @@
+# 🆔 Create identity
+
+By default, CryptKeeper uses MetaMask to create a new Semaphore Identity \[link].
+
+1. In MetaMask, a user signs a message with the private key of their Ethereum account.
+2. In your dapp, the user creates a deterministic identity \[link] with the signed message.
+3. The user can now recreate their Semaphore identity whenever they want by signing the same message with their Ethereum account in Metamask.
+
+Semaphore identities can also be created using other data (such as web2 reputations from [Interep](https://interep.link/)) or using random numbers.
+
+[@src/background/zk-keeper.ts](https://github.com/privacy-scaling-explorations/crypt-keeper-extension/blob/master/src/background/zk-keeper.ts)
+
+```typescript
+this.add(
+            RPCAction.CREATE_IDENTITY,
+            LockService.ensure,
+            async (payload: NewIdentityRequest) => {
+                try {
+                    const { strategy, messageSignature, options } = payload
+                    if (!strategy) throw new Error('strategy not provided')
+
+                    const numOfIdentites = await this.identityService.getNumOfIdentites()
+                    const config: any = {
+                        ...options,
+                        name: options?.name || `Account # ${numOfIdentites}`
+                    }
+
+                    if (strategy === 'interrep') {
+                        console.log("CREATE_IDENTITY: 1")
+                        config.messageSignature = messageSignature;
+                        console.log("CREATE_IDENTITY: 2")
+                    }
+
+                    const identity: ZkIdentityWrapper | undefined = await identityFactory(strategy, config)
+                    console.log("CREATE_IDENTITY: 4", identity);
+
+                    if (!identity) {
+                        throw new Error('Identity not created, make sure to check strategy')
+                    }
+
+                    await this.identityService.insert(identity)
+
+                    return true
+                } catch (error: any) {
+                    console.log("CREATE_IDENTITY: Error", error);
+                    throw new Error(error.message)
+                }
+            }
+        )
+```
```

Comment:
I don't think users need this code and documentation how it works under the hood.